buildpack/telemetry/metrics.py

```diff
@@ -170,7 +170,9 @@ def configure_metrics_registry(m2ee):
 
     paidapps_registries = {"Metrics.Registries": [INFLUX_REGISTRY]}
 
-    if datadog.is_enabled() or get_appmetrics_target():
+    appd_enabled = appdynamics.machine_agent_enabled()
```

Comment:
this is confusing. appd_enabled and machine agent enabled should be two different things. appd can be enabled with only java agent as well. Lets make it clear in code.

Reply:
also use appdynamics in variable names and not appd for consistency

Reply:
Removed this variable